seed/challenges/object-oriented-and-functional-programming.json

```diff
@@ -83,103 +89,163 @@
         "",
         "};",
         "",
+        "// Only change code above this line.",
+        "",
         "var myMotorBike = new MotorBike();",
         "",
-        "// Only change code above this line.",
         "",
         "(function() {return JSON.stringify(myMotorBike);})();"
       ],
+      "solutions":[
+        "var Car = function() {\n  this.wheels = 4;\n  this.engines = 1;\n  this.seats = 1;\n};\n\nvar myCar = new Car();\n\nvar MotorBike = function() {\n  this.engines = 1;\n  this.seats = 1;\n  this.wheels = 4;\n};\n\nvar myMotorBike = new MotorBike();\n\n(function() {return JSON.stringify(myMotorBike);})();\n"
+      ],
       "challengeType":1,
```

Comment:
Solutions shouldn't have the last bit of code `\n\n(function() { return JSON.stringify(myMotorBike);})();/n` just the solution themselfs